src/neo-asm/ASM_Language/linker/BuildedModule.cs

```diff
@@ -0,0 +1,122 @@
+using Newtonsoft.Json.Linq;
```

Comment:
THank you for using a standard JSON library instead of one of the two home grown libraries used in NEO and NEON